### PR TITLE
Skeleton Records Bugs

### PR DESCRIPTION
**Pull request by ChazUK**
Adding Start Screen for Basic Records
Save and Return for Total Staff

### CODE REVIEW

**Comment by ChazUK** [comment]
[Workplace dashboard showing wrong info for New starters](https://trello.com/c/CnaJ37pP/560-workplace-dashboard-showing-wrong-info-for-new-starters)

**Comment by ChazUK** [comment]
[User should be able to update Total Staff number from the dashboard without entering more staff](https://trello.com/c/lqifxtmU/562-user-should-be-able-to-update-total-staff-number-from-the-dashboard-without-entering-more-staff)

**Comment by ChazUK** [comment]
[Full record flow doesn't show the start page.](https://trello.com/c/iKaJteGH/561-full-record-flow-doesnt-show-the-start-page)

File: src/app/features/workers/create-basic-records-start-screen/create-basic-records-start-screen.component.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  <h1 class="h1">Staff records</h1>
+  <p>To create staff records for your workplace, for each record you will need to add their:</p>
+  <ul class="list list--bullet">
+    <li>full name or ID number</li>
+    <li>main job role</li>
+    <li>type of contract</li>
+  </ul>
+  <p>Once you have added these you will be able to continue with each record.</p>
+  <p>
+    We will use this information to work out your total number of staff. You will be able to change the number later by
```
Comment (wozzer72) [inline src/app/features/workers/create-basic-records-start-screen/create-basic-records-start-screen.component.html]: Isn't it a little confusing now to say "we will use this information to work out your total number of staff"? We're no longer updating total staff, so what does "work out" really mean?

File: src/app/features/workers/total-staff/total-staff.component.ts [hunk 2]
```diff
@@ -11,16 +12,22 @@ import { Subscription } from 'rxjs';
...
     private messageService: MessageService,
-    private establishmentService: EstablishmentService
+    private establishmentService: EstablishmentService,
+    private workerService: WorkerService
   ) {}
 
   ngOnInit() {
+    this.returnToDash = this.workerService.totalStaffReturn;
+
+    console.log(this.returnToDash);
```
Comment (wozzer72) [inline src/app/features/workers/total-staff/total-staff.component.ts]: remove the console debug

File: src/app/core/services/worker.service.ts [hunk 3]
```diff
@@ -65,6 +66,10 @@ export class WorkerService {
     return this.returnToSummary$.value as boolean;
   }
 
+  public get totalStaffReturn() {
+    return this.totalStaffReturn$.value as boolean;
```
Comment (wozzer72) [inline src/app/core/services/worker.service.ts]: This is a frontend value not an API value yes? When I first started to review this I thought it was a wrapper for Establishment number of staff.

But this is actually the count of worker records I believe having reviewed the rest of the changes below.
In mathematics, "total" is the sum of something, hence "total staff" suggest the "sum of staff".

Perhaps a name more like "currentNumberOfStaff"? Or "countOfStaff"?

Also, what is the suffix "Return" implying?

I know this is tedious reasoning, but great value comes from the name of attributes/methods, especially when there are two similar values (that declared and call upon via the API for establishment and that being the count of Worker records).

File: src/app/features/workers/create-basic-records-start-screen/create-basic-records-start-screen.component.html (hunk 1 above)
Comment (ChazUK) [reply src/app/features/workers/create-basic-records-start-screen/create-basic-records-start-screen.component.html]: Waiting on copy from Charlie

File: src/app/core/services/worker.service.ts (hunk 3 above)
Comment (ChazUK) [reply src/app/core/services/worker.service.ts]: This is just a boolean value that dictates wether or not the totalStaff should return to the dashboard, yes it's pretty badly named, but I want to review how this kind of thing works in the updated GDS branch as we're using this in a few other places.

File: src/app/features/workers/total-staff/total-staff.component.ts (hunk 2 above)
Comment (ChazUK) [reply src/app/features/workers/total-staff/total-staff.component.ts]: Removed

**Pull request merged**